Description: Fix warning about deprecated macro _BSD_SOURCE 
 use _DEFAULT_SOURCE: _BSD_SOURCE and _SVID_SOURCE are deprecated since
 glibc 2.20
Author: Sophie Brun <sophie@freexian.com>
Last-Update: 2016-03-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- configure
+++ configure
@@ -1040,7 +1040,7 @@ fi
 
 case "$target_os" in 
 linux*)
-	DEFINES="$DEFINES -D_BSD_SOURCE=1 -DLINUX"
+	DEFINES="$DEFINES -D_DEFAULT_SOURCE=1 -DLINUX"
 	;;
 solaris*)
 	LIBS="$LIBS -lsocket -lnsl"
--- configure.in
+++ configure.in
@@ -50,7 +50,7 @@ AC_CHECK_LIB(m, pow)
 dnl System-specific stuff
 case "$target_os" in 
 linux*)
-	DEFINES="$DEFINES -D_BSD_SOURCE=1 -DLINUX"
+	DEFINES="$DEFINES -D_DEFAULT_SOURCE=1 -DLINUX"
 	;;
 solaris*)
 	LIBS="$LIBS -lsocket -lnsl"
